jsk_arc2017_baxter/euslisp/lib/baxter-interface.l

```diff
@@ -15,17 +16,18 @@
 (defclass jsk_arc2017_baxter::baxter-interface
   :super baxter-interface
   :slots (rarm-pressure-threshold-
-          larm-pressure-threshold-))
+          larm-pressure-threshold-
+          right-hand-action-))
 
 (defmethod jsk_arc2017_baxter::baxter-interface
   (:init
     (&rest args &key
            ((:moveit-environment mvit-env)
-            (instance jsk_2016_01_baxter_apc::baxter-moveit-environment))
-           ((:moveit-robot mvit-rb) (instance jsk_2016_01_baxter_apc::baxter-robot :init))
+            (instance jsk_arc2017_baxter::baxter-moveit-environment))
```

Comment:
Ok, I'll add `(require "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l")` in `baxter-interface.l` to avoid namespace error, so please remove that in #2145